src/vs/workbench/contrib/customEditor/browser/customEditors.ts

```diff
@@ -117,7 +118,7 @@ export class CustomEditorService extends Disposable implements ICustomEditorServ
 						id: contributedEditor.id,
 						label: contributedEditor.displayName,
 						detail: contributedEditor.providerDisplayName,
-						describes: (currentEditor) => currentEditor instanceof CustomEditorInput && currentEditor.viewType === contributedEditor.id,
+						describes: editorId => editorId === contributedEditor.id,
```

Comment:
It now seems to me that every implementation of `describes` is comparing whether the passed in `editorId` matches the contributed editor ID. Isn't that something we could do from the outside because the editor is already contributed with an identifier? Why does the contributor here need to make this check?

Reply:
Good catch. It most likely is, let me go ahead and internalize this check